### PR TITLE
Move readers to fairmat-readers-xrd: Make changes in measurements

### PR DESCRIPTION
**Pull request by ka-sarthak**
- [x] remove the readers that have been moved to `fairmat-readers-xrd` https://github.com/FAIRmat-NFDI/readers-xrd/pull/1
- [x] remove the related utils and tests
- [x] add dependency on `fairmat-readers-xrd`

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -26,6 +26,7 @@ classifiers = [
 dependencies = [
     "nomad-lab>=1.2.2.dev399",
     "xmltodict==0.13.0",
+    "fairmat-readers-xrd~=0.0.3"
```
Comment (hampusnasstrom) [inline pyproject.toml:29]: Why the approx sign?

**Review by hampusnasstrom — COMMENTED** [review]
Not sure about the dependency version. 
Also, do you keep any tests of the normalize functions? Was wondering since you removed the example files.

File: pyproject.toml (hunk 1 above)
Comment (ka-sarthak) [reply pyproject.toml:29]: this way we can specify to use a compatible version with 0.0.3, which will be the latest version of the form: `0.0.*`. Ideally, the newer version of this form should be backward compatible.
https://peps.python.org/pep-0440/#compatible-release

**Comment by ka-sarthak** [comment]
> Also, do you keep any tests of the normalize functions? Was wondering since you removed the example files.

@hampusnasstrom Ah the data I remove are reference `json` files to compare the output from the readers.

The test for normalize functions in available in `test_parser.py`. This could be relocated though.

File: pyproject.toml (hunk 1 above)
Comment (hampusnasstrom) [reply pyproject.toml:29]: Ah, cool I didn't know this. Thanks for the info :)

**Review by hampusnasstrom — APPROVED** [review]

**Pull request merged**